src/main/java/net/tardis/mod/client/guis/GuiOptifineWarning.java

```diff
@@ -61,12 +60,17 @@ public void drawScreen(int mouseX, int mouseY, float partialTicks) {
 		@SubscribeEvent
 		public static void onLoadMC(GuiScreenEvent.InitGuiEvent e) {
 			if (!opened && FMLClientHandler.instance().hasOptifine() && e.getGui() instanceof GuiMainMenu) {
-				lastGui = e.getGui();
-				Minecraft.getMinecraft().displayGuiScreen(new GuiOptifineWarning());
-				opened = true;
+				{
+					if (TardisConfig.MISC.warn==true) {
```

Comment:
Why `if (TardisConfig.MISC.warn==true)` and not just `if (TardisConfig.MISC.warn)`

Reply:
Didn't know i could i'm new to trying java.